libs/message-handler/src/protocols/forward-client.ts

```diff
@@ -0,0 +1,89 @@
+import {
+  DIDDoc,
+  DIDResolver,
+  IMessage,
+  Message,
+  SecretsResolver,
+} from 'didcomm';
+import { v4 as uuidv4 } from 'uuid';
+import axios from 'axios';
+import PeerDIDResolver from '../../../did-resolver-lib/src/resolver';
```

Comment:
to import this as a package without specifying the part follow this steps
add this to did-resolver-libs/src/index.ts
```
export {
  ExampleDIDResolver,
  ExampleSecretsResolver,
} from './ExampleDIDResolver';
export { default as PeerDIDResolver } from './resolver';
```
then in the package.json of the did-resolver-lib add this
```
  "main": "src/index.ts" 
  ```
  then in the message-handler package.json add
  ```"did-resolver-lib": "*"```

Reply:
it ia already done